crates/orchestrator/src/tests/jobs/mod.rs

```diff
@@ -130,19 +133,20 @@ async fn create_job_job_handler_is_not_implemented_panics() {
 
     // Queue checks.
     let consumed_messages =
-        services.config.queue().consume_message_from_queue(JOB_PROCESSING_QUEUE.to_string()).await.unwrap_err();
+        services.config.queue().consume_message_from_queue(PROVING_JOB_PROCESSING_QUEUE.to_string()).await.unwrap_err();
     assert_matches!(consumed_messages, QueueError::NoData);
 }
 
 /// Tests `process_job` function when job is already existing in the db and job status is either
 /// `Created` or `VerificationFailed`.
 #[rstest]
-#[case(JobType::SnosRun, JobStatus::Created)]
-#[case(JobType::DataSubmission, JobStatus::VerificationFailed)]
+#[case(JobType::SnosRun, JobStatus::Created, SNOS_JOB_VERIFICATION_QUEUE)]
+#[case(JobType::DataSubmission, JobStatus::VerificationFailed, DATA_SUBMISSION_JOB_VERIFICATION_QUEUE)]
```

Comment:
nitpick: I don't think we can use the Queue_Name here, 
wdyt @byteZorvin ?

Reply:
Would want to to keep the case with specific values, personal preference though